testing/tests/DevExpress.ui.widgets.treeList/treeList.tests.js

```diff
@@ -1570,7 +1570,6 @@ QUnit.module('Scroll', defaultModuleConfig, () => {
         setTimeout(function() {
         // assert
             assert.strictEqual(treeList.pageIndex(), 2, 'page index');
-            assert.strictEqual(contentReadySpy.callCount, 3, 'onContentReady');
```

Comment:
Can we remove `contentReadySpy` now?

Reply:
since it is not used in the test, I think we can:)